proto/ca_service.proto

```diff
@@ -15,7 +15,14 @@
 syntax = "proto3";
 import "google/rpc/status.proto";
 
-package istio.v1.auth;
+package proto;
```

Comment:
you should only change go_package option for generating go codes. changing package here will break the protocol.